code/__DEFINES/DNA.dm

```diff
@@ -95,6 +95,10 @@
 #define ORGAN_SLOT_APPENDIX "appendix"
 #define ORGAN_SLOT_RIGHT_ARM_AUG "r_arm_device"
 #define ORGAN_SLOT_LEFT_ARM_AUG "l_arm_device"
+#define ORGAN_SLOT_RIGHT_HAND "r_hand"
+#define ORGAN_SLOT_LEFT_HAND "l_hand"
+#define ORGAN_SLOT_RIGHT_LEG "r_leg"
+#define ORGAN_SLOT_LEFT_LEG "l_leg"
```

Comment:
These don't appear to be used anywhere? what are they doing in here anyway

Reply:
aaaa i literally mentioned that earlier i can remove if you'd like but there are plans for them

Reply:
no one said anything to me about them for four days

Reply:
If you want a timely answer and there isn't obviously a maintainer keeping an eye on the pr you should get on the irc and ask one of us there.

In short though you should remove this if you're not using it here.